### PR TITLE
Update "locale" module docs

### PR DESCRIPTION
**Pull request by alessandrococco**
~~This PR adds documentation for the methods added/updated in https://github.com/espruino/BangleApps/pull/1729~~

### CODE REVIEW

**Comment by alessandrococco** [comment]
Please do not merge yet, I made some changes to the original PR (now it contains only the bug fix)

File: tutorials/Bangle.js Locale.md [hunk 1]
```diff
@@ -16,108 +16,122 @@ into Bangle.js's firmware, but you can install the `Language` app from [the App
...
 
 // Date to meridian (text describing morning/evening)
->require('locale').medidian(new Date())
+>require("locale").medidian(new Date())
 ="" // or "pm" for en_GB
 ```
 
-Translation
-------------
+Translations (deprecated)
```
Comment (gfwilliams) [inline tutorials/Bangle.js Locale.md:130]: Might be worth mentioning  `/*LANG*/` here as the non-deprecated option

Comment (alessandrococco) [reply tutorials/Bangle.js Locale.md:130]: Ok! I can add some info about the `LANG` stuff :)

Comment (alessandrococco) [reply tutorials/Bangle.js Locale.md:130]: I'll add some documentation about the new `date_utils` also.

**Comment by gfwilliams** [comment]
Looks great - thanks!

**Pull request merged**